src/diffpy/__init__.py

```diff
@@ -12,3 +12,13 @@
 # See LICENSE.rst for license information.
 #
 ##############################################################################
+
```

Comment:
per convo with @bobleesj this block can be deleted so (apart from the comments with the copyright statement) this file should be empty.